src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/MinimumWageCalculator.cs

```diff
@@ -6,36 +6,32 @@ public class MinimumWageCalculator : IMinimumWageCalculator
     {
         private const decimal WeeksPerMonth = 4m;
         private const decimal WeeksPerYear = 52m;
-        private const string MissingMinWageErrorMessage = "MinWage can't be null.";
         private const string HoursPerWeekZeroErrorMessage = "HoursPerWeek must be greater than 0.";
         private const string IncorrectWageTypeErrorMessage = "WageUnit must be either 'Weekly', 'Monthly' or 'Annually'.";
 
-        public decimal CalculateMinimumWage(CreateApprenticeshipRequest request)
+        public decimal CalculateMinimumWage(decimal wageValue, WageUnit wageUnit, decimal hoursPerWeek)
         {
-            if (!request.MinWage.HasValue)
-                throw new ArgumentOutOfRangeException(nameof(request.MinWage), request.MinWage, MissingMinWageErrorMessage);
-
-            if (!(request.HoursPerWeek > 0))
-                throw new ArgumentOutOfRangeException(nameof(request.HoursPerWeek), request.HoursPerWeek, HoursPerWeekZeroErrorMessage);
+            if (!(hoursPerWeek > 0))
```

Comment:
Suggest we change this to positive validation `if (hoursPerWeek <= 0)`

Reply:
Changed